infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/StreamsSnapshotReader.java

```diff
@@ -54,10 +54,11 @@ public class StreamsSnapshotReader implements SnapshotReader {
      */
     private final int maxDataSizePerMsg;
     private final Optional<DistributionSummary> messageSizeDistributionSummary;
+    private final CorfuRuntime rt;
+    private final LogReplicationConfig config;
     private long snapshotTimestamp;
     private Set<String> streams;
```

Comment:
Update the class-level comment adding that the list of streams to replicate will get synced with the registry table and refreshed at the start of a snapshot sync.

Reply:
did not see this comment?